scalableminds/webknossos
# upgrade grpc and use new fossildb standard health check

### note: needs datastore upgrade

- use the new fossilDB that comes with the standardized grpc health check
- use that health check
- add checking it to the datastore’s own health endpoint
- use new versions of protobuf/grpc dependencies (package structure changed)
- get rid of the scalapb-json4s dependency (as we are not sending protobuf as json anymore since #2650)

### Steps to test:
- Set up: [Here is how to set up a standalone-datastore locally](https://github.com/scalableminds/webknossos-wiki/wiki/Set-up-a-standalone-datastore-locally)
- Create a new task & try to get a new task --> should work
- Get another new task -> should fail with no datastore-related error message
- Open a dataset --> should work
- Kill the fossildb
- Create a new task & try to get a new task --> should fail with a warning about the datastore
- Open a dataset --> shouldn't display data (but instead warn about the datastore)


### Issues:
- contributes to #2604

------
